### PR TITLE
Rdk 34538 maintenance manager plugin document update

### PR DESCRIPTION
**Pull request by s-keerthi**
Documentation enhancement for Maintenance Manager plugin
Jira link:  https://ccp.sys.comcast.net/browse/RDK-34538

### CODE REVIEW

File: MaintenanceManager/MaintenanceManager.json [hunk 1]
```diff
@@ -31,15 +31,15 @@
...
             "result": {
                 "type": "object",
                 "properties": {
                     "maintenanceStatus":{
                         "$ref": "#/definitions/maintenanceStatus" 
                     },
-                    "lastSuccessfulCompletionTime":{
-                        "summary": "The time the last maintenance completed or `0` if not applicable",
+                    "LastSuccessfulCompletionTime":{
+                        "summary": "The time (in seconds) the last maintenance completed or `0` if not applicable",
```
Comment (livinsunny519) [inline MaintenanceManager/MaintenanceManager.json:42]: (in Epoch time)

File: MaintenanceManager/MaintenanceManager.json [hunk 2]
```diff
@@ -59,20 +59,20 @@
...
         },
         "getMaintenanceStartTime":{
-            "summary": "Gets the scheduled maintenance start time",
+            "summary": "Gets the scheduled maintenance start time. \n \n### Events\n \n No Events.",
             "result": {
                 "type": "object",
                 "properties": {
                     "maintenanceStartTime": {
-                        "summary": "The start time",
+                        "summary": "The start time (in seconds)",
```
Comment (livinsunny519) [inline MaintenanceManager/MaintenanceManager.json:75]: (in epoch time)

File: MaintenanceManager/MaintenanceManager.json [hunk 3]
```diff
@@ -87,12 +87,12 @@
...
         },
         "setMaintenanceMode":{
-            "summary": "Sets the maintenance mode",
+            "summary": "Sets the maintenance mode. \n \n### Events\n \n No Events.",
             "params": {
                 "type":"object",
                 "properties": {
                     "maintenanceMode":{
-                        "summary": "The maintenance mode. The `FOREGROUND` mode runs all maintenance tasks. The `BACKGROUND` mode runs maintenance tasks that do not impact the user experience.",
+                        "summary": "The maintenance mode. The `FOREGROUND` mode runs all maintenance tasks in the foreground. The `BACKGROUND` mode runs maintenance tasks in the background that do not impact the user experience.",
```
Comment (livinsunny519) [inline MaintenanceManager/MaintenanceManager.json:95]: The `FOREGROUND` mode runs all maintenance tasks.
BACKGROUND-
    Abort activities currently running and if the task can't run in the background (if maintenance was already started with FOREGROUND mode) 
    Execute tasks not impacting user experience if the maintenance mode is set to BACKGROUND before calling startMaintenance

File: MaintenanceManager/doc/MaintenanceManagerPlugin.md [hunk 4]
```diff
@@ -112,7 +125,7 @@ This method takes no parameters.
 | :-------- | :-------- | :-------- |
 | result | object |  |
 | result.maintenanceStatus | string | The current maintenance status |
-| result.lastSuccessfulCompletionTime | integer | The time the last maintenance completed or `0` if not applicable |
+| result.LastSuccessfulCompletionTime | integer | The time (in seconds) the last maintenance completed or `0` if not applicable |
```
Comment (livinsunny519) [inline MaintenanceManager/doc/MaintenanceManagerPlugin.md:128]: (in epoch time)

File: MaintenanceManager/MaintenanceManager.json (hunk 1 above)
Comment (s-keerthi) [reply MaintenanceManager/MaintenanceManager.json]: Modified into epoch time

File: MaintenanceManager/MaintenanceManager.json (hunk 2 above)
Comment (s-keerthi) [reply MaintenanceManager/MaintenanceManager.json]: Modified into epoch time

File: MaintenanceManager/MaintenanceManager.json (hunk 3 above)
Comment (s-keerthi) [reply MaintenanceManager/MaintenanceManager.json]: Added the given description

File: MaintenanceManager/doc/MaintenanceManagerPlugin.md (hunk 4 above)
Comment (s-keerthi) [reply MaintenanceManager/doc/MaintenanceManagerPlugin.md]: Modified itno epoch time

**Review by livinsunny519 — APPROVED** [review]

**Pull request merged**